packages/components/package.json

```diff
@@ -26,34 +26,21 @@
     "lint:typescript": "../../scripts/check-typescript"
   },
   "devDependencies": {
-    "@babel/runtime": "^7.12.5",
-    "date-fns": "^2.16.1",
     "nebenan-eventproxy": "^1.0.0",
     "nebenan-form": "^10.0.0",
-    "nebenan-helpers": "^7.1.0",
     "nebenan-keymanager": "^1.0.1",
-    "nebenan-ui-kit": "^5.1.0",
-    "node-sass-glob-importer": "^5.3.2",
-    "prop-types": "^15.7.2",
-    "qs": "^6.9.4",
-    "react": "^16.14.0",
-    "react-dom": "^16.14.0",
     "react-load-script": "^0.0.6",
     "react-redux": "^7.2.2",
-    "react-router": "^5.2.0",
```

Comment:
are you sure we don't need those? i think we're using react-router in preview

Reply:
those devDependencies are all listed in the root package-json